LoopKit/AnyCodableEquatable.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  AnyCodableEquatable.swift
+//  LoopKit
+//
+//  Created by Darin Krauss on 2/8/22.
+//  Copyright © 2022 LoopKit Authors. All rights reserved.
+//
+
+import Foundation
+
+public struct AnyCodableEquatable: Codable, Equatable {
+    public enum Error: Swift.Error {
+        case unknownType
+    }
+
+    public let any: Any
```

Comment:
[nit] A variable named `any`, while correct, I guess, seems a little strange to me.  Maybe consider just `value`?  Another common name is `wrapped`.  Up to you.

Reply:
Changed to `wrapped`.